wgpu-core/platform-features/wasm/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "wgpu-core-platform-wasm"
```

Comment:
[not actionable] _technically_ this crate is about `wasm32` right now. But sure enough once wasm64 (is that what it's called?) becomes more mainstream this should encompass that too 🤔

Reply:
Yeah no reason we couldn't expand that this crate when needed - I think it's better to be broad with the name so we don't end up with two crates on crates-io..